.github/workflows/push-to-main.yaml

```diff
@@ -49,6 +50,45 @@ jobs:
           username: ${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_USERNAME }}
           password: ${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_PASSWORD }}
 
+  publish-flightctl-ocp-ui:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Generate tags
+        id: generate-tags
+        run: |
+          tags=( latest-${GITHUB_SHA} latest )
+          echo "tags=${tags[@]}" >> $GITHUB_OUTPUT
+          echo "tags=${tags[@]}"
+
+      - name: Build
+        id: build
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: ${{ env.QUAY_OCP_REPO }}
```

Comment:
Should we rename `QUAY_REPO` to `QUAY_STANDALONE_REPO`?

Reply:
I dont have a strong preference for one over the other. Do you ?

Reply:
I'm fine with the current name. I think since we have different repositories, better to add an identifier to each one.

Reply:
ok, renamed